node/engine/parse/postgres/parse.go

```diff
@@ -15,7 +15,7 @@ func doNothing(_ string) error {
 
 // CheckSyntaxReplaceDollar replaces all bind parameters($x) with 1 to bypass
 // syntax check errors.
-// ToSQL() method doesn't convert bind parameters to $1, $2, etc. so we need to
+// () method doesn't convert bind parameters to $1, $2, etc. so we need to
```

Comment:
Hmm, was this obsoleted?

Reply:
Yep, I think that got screwed up via a mass replace, but yes this package is actually not used anymore